test/modules/uiTheme/components/toggle/ThemeToggle.test.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-undef */
+import { ThemeToggle } from '../../../../../src/modules/uiTheme/components/toggle/ThemeToggle';
+import { TestUtils } from '../../../../test-utils';
+import { uiThemeReducer } from '../../../../../src/modules/uiTheme/store/uiTheme.reducer';
+import { $injector } from '../../../../../src/injection';
+
+window.customElements.define(ThemeToggle.tag, ThemeToggle);
+
+let store;
+
+describe('ThemeToggle', () => {
+
+	const setup = () => {
+		const state = {
+			uiTheme: {
+				theme: 'dark'
+			}
+		};
+		store = TestUtils.setupStoreAndDi(state, {
+			uiTheme: uiThemeReducer
+		});
+
+		$injector
+			.registerSingleton('TranslationService', { translate: (key) => key });
+
+		return TestUtils.render(ThemeToggle.tag);
+	};
+
+	describe('when initialized', () => {
+		it('renders the view', async () => {
+			const element = await setup();
+
+			expect(element.shadowRoot.querySelector('.switch').title).toBe('uiTheme_toggle_tooltip_dark');
+			expect(element.shadowRoot.querySelector('input').checked).toBeTrue();
+			expect(element.shadowRoot.querySelector('.icon.adjust')).toBeTruthy();
+			expect(element.shadowRoot.querySelector('.slider.round')).toBeTruthy();
+		});
+	});
```

Comment:
all tests here in a wellformed AAA-Manner /arrange/act/assert. 

Do we give tests the structure-comments 
`// arrange
// act 
// assert`
?

Reply:
Not when it's clear what's happening:
https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80